python/python/glide/async_commands/cluster_commands.py

```diff
@@ -353,6 +353,58 @@ async def function_load(
             ),
         )
 
+    async def function_list(
+        self,
+        library_name_pattern: Optional[str] = None,
+        with_code: bool = False,
+        route: Optional[Route] = None,
+    ) -> List[
+        Mapping[bytes, Union[bytes, List[Mapping[bytes, Union[bytes, Set[bytes]]]]]]
```

Comment:
For this command the routing arg can change the response type depending on what route is provided, which you can represent by wrapping this in TClusterResponse. Let's see how big we can make this return type (=
```suggestion
    ) -> TClusterResponse[List[
        Mapping[bytes, Union[bytes, List[Mapping[bytes, Union[bytes, Set[bytes]]]]]]]
```